src/components/Instructions/GettingStarted/GettingStarted.js

```diff
@@ -813,7 +929,7 @@ const GettingStarted = ({ config }) => (
             GitHub respository). You can also add additional files and publications,
             and
             {' '}
-            <a href="../examples/alpheios-integration/index.html" target="_blank" rel="noopener noreferrer">
+            <a href="../../examples/alpheios-integration" target="_blank" rel="noopener noreferrer">
               integrate it with Alpheios
             </a>
```

Comment:
Maybe it would be good to add a small paragraph to the end here about updating? I like the menu at the top but I didn't notice it when I was reading the page.

So something like "The Perseids Treebank Template code gets updated with new features from time to time.  We have created a workflow that makes it easy to pull the updates into your repository.  See the <a href...> Updating</a> instructions for details.

What do you think?

Reply:
That's a great idea. I've updated the text and also fixed an issue with relative links.